Zotero/Scenes/Detail/PDF/Views/PDFReaderViewController.swift

```diff
@@ -421,6 +421,13 @@ class PDFReaderViewController: UIViewController {
     // MARK: - Actions
 
     private func update(state: PDFReaderState) {
+        if state.changes.contains(.md5) {
+            self.coordinatorDelegate?.showDocumentChangedAlert(key: state.key, parentKey: state.parentKey, libraryId: state.library.identifier) { [weak self] in
```

Comment:
Remove `self`.

Reply:
Removed in 9b5ebbbe2e66e8aa4763a6155777bfb0053b416e